File: legacy/legacy-apis/Psi.Models.ClientConfigurationModels/Admin/MfaSettings.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Psi.Data.Models.ClientConfigurationModels.Admin
{
    public class MfaSettings : SettingsBaseHelper
    {
        public MfaSettings(ISettingsBase settingsBase) : base(settingsBase)
        {
        }

        /// <summary>
        /// If set to True, mfa text questions are enabled
        /// </summary>
        [SettingKey("Admin.MFA.MfaTextQuestionsEnabled")]
        public bool MfaTextQuestionsEnabled
        {
            get { return GetBoolValue(); }
            set { SetValue(value); }
        }
    }
}

File: workspace/legacy-analyzer/output/infrastructure/models/Psi/Data/Models/ClientConfigurationModels/Admin/MfaSettings.ts
import { Setting, ISettingsGroup, ISettingsMetadata } from '@models/base/types';
export interface MfaSettingsConfig {
    MfaTextQuestionsEnabled: boolean;
}

export class MfaSettings implements ISettingsGroup {
    static readonly metadata: ISettingsMetadata = {
        groupName: 'MfaSettings'
    };


            private _mfaTextQuestionsEnabled: boolean;
            get mfaTextQuestionsEnabled(): boolean {
                return this._mfaTextQuestionsEnabled;
            }
            set mfaTextQuestionsEnabled(value: boolean) {
                this._mfaTextQuestionsEnabled = value;
            }


    constructor() {}


        toSettings(): Setting[] {
            return [
                { key: "MfaSettings.MfaTextQuestionsEnabled", value: this._mfaTextQuestionsEnabled, dataType: 'boolean', label: "Mfa Text Questions Enabled" },
            ];
        }

}